README.md

```diff
@@ -1,172 +1,124 @@
-
-  ```bash
-  pip install --upgrade pip
-  ```
-
-  To install the project's required packages (whenever dependencies must be
-  updated):
-
-  ```bash
-  pip install -r requirements.txt
-  ```
-
-* We follow
-  [Google's style guide](https://google.github.io/styleguide/pyguide.html).
-  Specifically, we format code with `yapf`, and check for bugs and style
-  problems with `pylint`.
+```
+.
+└── data_set/
+    ├── sim_1/
+    │   └── sim.vtk
+    └── sim_2/
+        └── sim.vtk
+```
 
-  Usually, there's no need to run these tools manually. We added continuous
-  integration (CI) through GitHub, so every time your pull request has errors
-  you'll be alerted via email. Over time you'll get used to these rules and
-  coding consistently will become second nature.
+The folder and file names can change.
 
-  However, if you want to run `yapf` and `pylint` locally, simply install them
-  via `pip`:
+If you do not have your own data you can download one of our datasets
+[here](https://drive.google.com/u/0/uc?id=1T5wXz-igXjrztLt6xgb4CZx4tGwXK8DX&export=download). This
+particular dataset has 1000 examples of objects:
 
-  ```bash
-  pip install yapf pylint
-  ```
+![alt_image](readme_images/dataset_example.png)
 
-* We write tests using `pytest`. See [`example_test.py`](example_test.py) for an
-  example and delete if example after you copied the template to your new
-  project.
+Actually, in the image, we can see an example of an object and the
+predictions of one of our models.
 
-  Tests are also run automatically in GitHub via continuous integration. If you
-  want to run them locally, install `pytest`:
 
-  ```bash
-  pip install pytest
-  ```
+The next step is to pre-process the dataset. To do this just run the
+script:
 
-  and run:
+```bash
+pytohn process --data_dir=<Path to your dataset>
+```
 
-  ```bash
-  pytest
-  ```
+You will see `.pt` objects appearing next to the `.vtk` objects. These
+are the torch graphs already constructed.
 
-  in your base directory.
+## Mlflow
 
-  `pytest` is smart enough to discover your tests automatically when you're
-  following
-  [the right conventions](https://docs.pytest.org/en/stable/goodpractices.html#conventions-for-python-test-discovery). At Inductiva we'd prefer that any
-  test file (e.g. `example_test.py`) stays in the same directory next to the
-  component it tests (e.g. `example.py`).
+We are using
+[mlflow](https://mlflow.org/docs/latest/python_api/mlflow.html) to
+keep track of our experiments. We have our own remote server to which
+we can log everything from any computer which was created following
+the instructions in [this
+tutorial](https://towardsdatascience.com/managing-your-machine-learning-experiments-with-mlflow-1cd6ee21996e).
 
-## Documentation
+This is a nice feature to have but it is not required in any way to
+run our code. Everything will be logged locally by default. That is,
+all experiments will be logged to the folder `mlruns` created in the
```

Comment:
Maybe change the default name of this folder. Is it possible?

Reply:
This is the mlflow default, it is not specified by us. The user would need to create an environment variable to change this.